tests/test_models_unet.py

```diff
@@ -199,7 +199,7 @@ def prepare_init_args_and_inputs_for_common(self):
 
     def test_gradient_checkpointing(self):
         init_dict, inputs_dict = self.prepare_init_args_and_inputs_for_common()
-        model = self.model_class(**init_dict)
+        model = self.model_class(**init_dict).eval()
```

Comment:
nice! Let's be extra careful in the future to not just lower the test tolerance. Here I think it really was dropout that was messing up the grads